server_manager/electron_app/build.action.sh

```diff
@@ -72,4 +72,4 @@ cd "${STATIC_DIR}"
 
 # Icons.
 cd "${ROOT_DIR}"
-electron-icon-maker --input=server_manager/images/launcher-icon.png "--output=${BUILD_DIR}/server_manager/electron_app/static"
+cp -r server_manager/electron_app/icons/ "${BUILD_DIR}/server_manager/electron_app/static/icons/"
```

Comment:
Will this change affect Windows or Linux build?

Reply:
No. Basically what I did was add the existing icons that `electron-icon-maker` made to the repo, then replaced the `mac/icon.icns` file with a new version.